quantum/split_common/transport.c

```diff
@@ -33,35 +64,38 @@ extern backlight_config_t backlight_config;
 
 // Get rows from other half over i2c
 bool transport_master(matrix_row_t matrix[]) {
-  i2c_readReg(SLAVE_I2C_ADDRESS, I2C_KEYMAP_START, (void *)matrix, ROWS_PER_HAND * sizeof(matrix_row_t), TIMEOUT);
+  i2c_readReg(SLAVE_I2C_ADDRESS, transport_values.matrix_start, (void *)matrix, ROWS_PER_HAND * sizeof(matrix_row_t), TIMEOUT);
```

Comment:
This isn't right and should be reverted back to I2C_KEYMAP_START.

Reply:
fixed